docs/HootenannyConfigOptions.asciidoc

```diff
@@ -58,7 +58,7 @@ hoot convert \
 
 The +$HOOT_HOME/conf/hoot.json+ configuration file is used with all commands without specifying the configuration on the command line.
 
-For +List+ type options, additional options can be appended to the list of defaults by using the ++=+ operator. E.g.
+For +List+ type options, additional options can be appended to the list of defaults by using the `+=` operator. E.g.
```

Comment:
Some places I changed + for ` for readability and here double pluses screwed up in the processing.